src/src/AdminBundle/Controller/EventController.php

```diff
@@ -36,32 +41,49 @@ public function createAction(Request $request)
     }
 
     /**
-     * @Route("/event/edit/{id}", name="event_edit")
+     * @Route("event_type/edit/{id}/event/edit/{event_id}", name="event_edit")
      * @Method({"GET", "POST"})
      */
-    public function editAction()
+    public function editAction($id, $event_id, Request $request)
     {
-        return $this->render('AdminBundle:Event:index.html.twig');
+        $repository = $this->getDoctrine()->getRepository(Event::class);
+        $event = $repository->findOneBy(['id' => intval($event_id)]);
+
+        $form = $this->createForm(EventForm::class, $event);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($event);
+            $em->flush();
+
+            return $this->redirectToRoute('event_type_edit', ['id' => $id]);
+        }
+
+        return $this->render('AdminBundle:Event:edit.html.twig', array(
+            'form' => $form->createView(),
+        ));
     }
 
     /**
-     * @Route("/event/delete/{id}", name="event_delete")
+     * @Route("event_type/edit/{id}/event/delete/{event_id}", name="event_delete")
      * @Method("POST")
      */
-    public function deleteAction($id, Request $request)
+    public function deleteAction($id, $event_id, Request $request)
     {
         $em = $this->getDoctrine()->getManager();
-        $event = $em->getRepository(Event::class)->findOneBy(array('id' => $id));
+        $event = $em->getRepository(Event::class)->findOneBy(array('id' => intval($event_id)));
```

Comment:
ak budu requirements je mozne, ze to nebude treba prekonvertovat (niesom si isty)